web-stories.php

```diff
@@ -41,7 +41,7 @@
 }
 
 define( 'WEBSTORIES_VERSION', '1.1.1' );
-define( 'WEBSTORIES_DB_VERSION', '3.0.2' );
+define( 'WEBSTORIES_DB_VERSION', '3.0.4' );
```

Comment:
There was a 3.0.3 method that was never run. Was this by design @swissspidy ?

Reply:
Yikes! Definitely not. That wouldn't make sense.

It was correctly changed in #4759 (4c20a3a1fb2c52a5231c07c868299e41a4d0445b), so I don't know how this was reverted again?!

Reply:
Hotfix: #5667